### ISSUE TITLE
Proxy websockets in dev server

### ISSUE THREAD

**Issue by Ajayneethikannan** [IMGIITRoorkee/omniport-frontend#25]
The issue is that,
1. ws urls are proxied as 'http://' requests
2. not able to proxy ws connections in development without external libraries such react.io-connect

Solutions for these would be great.

**Comment by dhruvkb** [comment]
Just wanted to ask if you are sure about this? My memory is a little foggy but iirc, @pradumangoyal and I were testing websockets (with "ping-pong") on my development server and it was working fine.

**Comment by pradumangoyal** [comment]
@dhruvkb I think that was django template-based. The one you made initially?

**Comment by dhruvkb** [comment]
Yeah the very first one. Was that made in Django templates?

**Comment by Ajayneethikannan** [comment]
I think maybe there is some issue in the webpack config 😅, I remember @algomaster99 facing some issues with WebSocket proxy while working on fac app publish

**Comment by algomaster99** [comment]
@Ajayneethikannan yeah, I couldn't simply use `/ws/do-something`. Had to write the full absolute URL with `ws://` mentioned.
